README.md

```diff
@@ -34,20 +38,18 @@ https://github.com/udacity/CarND-Capstone/issues/147 We locally updated our requ
 * Real world testing (will be updated)
 See below
 
-* Suggestions
-I wish Virtual Workspace had internet access so that instead of uploading files it would be possible to clone a repo.
 
 ### Other Issues we ran into
 1. Requirements.txt was modified to point pillow to version 4.3 (see above).  We had an issue with cv.Bridge that was traced back to https://github.com/udacity/CarND-Capstone/issues/147.  This required upgrade to pillow version 4.3
-- We do want to call out that while you mention not to change the requirements.txt, you have used different kinds of settings yourself in the Virtual Workspace
-- This has caused a lot of confusion and wasted time
+  1. We do want to call out that while you mention not to change the requirements.txt, you have used different kinds of settings yourself in the Virtual Workspace
```

Comment:
add a list:
CUDA v9.0 not, v8.0 as required
Driver 4.15, not 3.XX not as required.

Reply:
The same point about Docker applies here too?  That this is an issue only for Docker or also for Workspace & native installation?